fiftyone/utils/torch.py

```
@@ -295,6 +297,9 @@ def __call__(self, d):
         """
         raise NotImplementedError("subclasses must implement __call__()")
 
+    def _load_media(self, d):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_load_media`

(ANN202)